nuxeo-features/nuxeo-platform-publisher/nuxeo-platform-publisher-core/src/main/java/org/nuxeo/ecm/platform/publisher/impl/service/PublisherServiceImpl.java

```diff
@@ -99,7 +99,9 @@
 
     @Override
     public void applicationStarted(ComponentContext context) {
-        if (TransactionHelper.startTransaction()) {
+        boolean started = TransactionHelper.isTransactionActive();
+
+        if (started || TransactionHelper.startTransaction()) {
```

Comment:
agree @jcarsique ,  fix is in the "started" condition in the `commitOrRollbackTransaction()` block. The `started` variable name is not well chosen.